### ISSUE TITLE
Unable to navigate to folder in with Quot Libet on MacOS

### ISSUE THREAD

**Issue by thalunil** [quodlibet/quodlibet#3403]
Steps to reproduce
------------------
With Quot Libet on MacOS i am unable to right click on the song to navigate to the corresponding folder with the Finder.
Quod Libet on Linux works flawlessly.

Expected Output
---------------
Navigation to the folder should be easily possible.

Test System
-----------

### Which version of Quod Libet?
4.3.0

### Which operating system 
MacOS 10.15.5

**Comment by urielz** [comment]
Some info that might help track down the issue: it's working fine for me on Quodlibet 4.3.0 on macOS 10.13.6

**Comment by minusf** [comment]
i think this is related to the full disk access in catalina. I have added Quod Libet to full disk access but i still cannot access external media and Download, Documents.

**Comment by mundschenk-at** [comment]
I think the app needs to include some configuration keys in its plist: https://developer.apple.com/documentation/bundleresources/information_property_list/protected_resources

**Comment by thalunil** [comment]
> Some info that might help track down the issue: it's working fine for me on Quodlibet 4.3.0 on macOS 10.13.6

What info do you think of?
OS + Quod Libet version is already there...

**Comment by declension** [comment]
I guess the point is it might be MacOS 10.14+ that breaks

**Comment by declension** [comment]
> I think the app needs to include some configuration keys in its plist: https://developer.apple.com/documentation/bundleresources/information_property_list/protected_resources

Thanks. Any ideas what keys though? Some of the ones under _Files and Folders_ ?

**Comment by mundschenk-at** [comment]
> > I think the app needs to include some configuration keys in its plist: https://developer.apple.com/documentation/bundleresources/information_property_list/protected_resources
> 
> Thanks. Any ideas what keys though? Some of the ones under _Files and Folders_ ?

Yes. While I don't have any experience with developing macOS applications, from the description I think `NSDesktopFolderUsageDescription`, `NSDocumentsFolderUsageDescription`, `NSDownloadsFolderUsageDescription`, `NSNetworkVolumesUsageDescription` and `NSRemovableVolumesUsageDescription` are prime candidates.

**Comment by mundschenk-at** [comment]
> I guess the point is it might be MacOS 10.14+ that breaks

Unless you have specific reports from Mojave users (10.14), I think those changes were introduced in Catalina (10.15). Big Sur (11.0) would of course be also affected.
